### ISSUE TITLE
Scrollbars for Windows desktop apps should be wider and be visible if there is content to scroll

### ISSUE THREAD

**Issue by csells** [flutter/flutter#36412]
## Use case

Create a new Flutter application for the desktop and run the following code:

```dart
class MyApp extends StatelessWidget {
  final title = 'Desktop Scrolling Demo';
  @override
  Widget build(BuildContext context) => MaterialApp(
      title: title,
      theme: ThemeData(primarySwatch: Colors.blue),
      home: Scaffold(
          appBar: AppBar(title: Text(title)),
          body: Scrollbar(child: ListView(children: [for (var i = 0; i < 100; ++i) Text('item #$i')]))));
}
```

![image](https://user-images.githubusercontent.com/2568253/61421256-1c614c00-a8bb-11e9-9166-f31943bd1296.png)

In this app, the scrollbar is only shown while scrolling, even though there is content to scroll. Further, the scrollbar is too skinny for easy dragging with a mouse (even if you could catch it...).
 
## Proposal
On a Windows desktop app, the scrollbar should always be visible if there is content to scroll and the thumb should be wider to enable mouse dragging. macOS convention matches the current behavior.

**Comment by tvolkert** [comment]
There's already an `isAlwaysShown` parameter to `Scrollbar`.  Sounds like we should make it default to true on Windows.

... and add a `thickness` parameter.  There's already a `thickness` variable at the lower levels - it's just not exposed.

**Comment by SahajRana** [comment]
@tvolkert @csells, It would be great for CupertinoScrollBar as well. It doesn't have `isAlwaysShown` parameter.

**Comment by Piinks** [comment]
Hey @csells! :)

It looks like the thickness property of Scrollbar is now exposed. I also noticed there are additional customization options on CupertinoScrollbar, so I think we can take this a bit further for our next iteration of the scrollbar and introduce a ScrollbarTheme. I have filed #68800 for tracking that.
As for the out-of-the-box scrollbar feature, one that is always shown by default for scroll views on desktop and web, https://github.com/flutter/flutter/issues/40107 is currently tracking this. 

I think we have coverage for everything here, so I am going to close this issue. If I've left something out, let me know and we can re-open this issue. Thanks!
